composer.json

```diff
@@ -3,7 +3,8 @@
     "type": "library",
     "license": "Apache-2.0",
     "require": {
-        "php": "^7.1"
+        "php": "^7.1",
+        "phan/phan": "^2.4"
```

Comment:
I think phan should be a require-dev dependency, not require. It's not needed for using our project, just developing for it.

Reply:
I was planning on having phan run as part of the CI, so that's why I was including this here.  Happy to change it if you aren't of the same opinion?